src/libslic3r/GCode.cpp

```diff
@@ -2558,9 +2558,6 @@ void GCode::_do_export(Print& print, GCodeOutputStream &file, ThumbnailsGenerato
     file.write(m_writer.update_progress(m_layer_count, m_layer_count, true)); // 100%
     file.write(m_writer.postamble());
 
-    file.write(m_writer.set_chamber_temperature(0, false));  //close chamber_temperature
```

Comment:
I plan to rewrite this part in the future.
For now, can you use support_chamber_temp_control to check whether we need turn off chamber heater instead of deleting it?